mlflow/utils/credentials.py

```diff
@@ -76,6 +77,32 @@ def login(backend="databricks"):
         )
 
 
+def _is_valid_databricks_host(host):
+    # Check if the host is a valid Databricks URL: 1) starts with https://. 2) has "databricks.com"
+    # in the domain. 3) No params/query/fragment in the URL.
+    if not host.startswith("https://"):
+        _logger.error(f"Host must start with 'https://', but received: {host}")
+        return False
```

Comment:
I don't think we should be doing our own validation of the Databricks host. That's the job of the Databricks SDK

Reply:
@chenmoneygithub Instead of reimplementing our own validation logic, which may or may not be correct, can we implement a timeout for the databricks python sdk login procedure? E.g. maybe time out after 10 seconds and print a warning that the URL / token may be invalid? That will resolve the confusion.

Reply:
discussed offline, sounds good!